lib/js/components/CounterToggle/CounterToggle.vue

```diff
@@ -0,0 +1,367 @@
+				'icon': $color-inverted-icon-disabled,
+				'background': $color-default-background-ghost-disabled,
+			),
+		),
+		'selected': (
+			'color': $color-primary-text,
+			'color-hovered': $color-primary-text-hovered,
+			'color-focused': $color-primary-text-focused,
+			'background': $color-primary-background,
+			'background-hovered': $color-primary-background-hovered,
+			'background-focused': $color-primary-background-focused,
+			'icon': $color-primary-icon,
+			'ripple': $color-primary-ripple,
+			'disabled': (
+				'color': $color-primary-text-disabled,
+				'icon': $color-primary-icon-disabled,
+				'background': $color-primary-background-disabled,
+			),
+		),
+	),
+);
+
+@mixin setCounterToggleColor(
+	$color,
+	$color-hovered: null,
+	$color-focused: null,
+	$color-disabled: null
+) {
+	color: $color;
+
+	&:hover,
+	&.-hovered,
+	&:active,
+	&.-active {
+		color: $color-hovered;
+	}
+
+	&:focus,
+	&.-focused {
+		color: $color-focused;
+	}
+
+	&.-disabled {
+		color: $color-disabled;
+	}
+}
+
+@mixin setCounterToggleBackground($background, $background-hover, $background-focus) {
+	@if $background == null {
+		background-color: transparent;
+	}
+
+	background-color: $background;
+
+	&:hover,
+	&.-hovered,
+	&:active,
+	&.-active {
+		background-color: $background-hover;
+	}
+
+	&:focus,
+	&.-focused {
+		background-color: $background-focus;
+	}
+}
+
+@mixin setCounterToggleDisabled(
+	$disabled-color: null,
+	$disabled-icon: null,
+	$disabled-background: null,
+	$disabled-border: null
+) {
+	&:disabled,
+	&.-disabled {
+		border-color: $disabled-border;
+		color: $disabled-color;
+		background-color: $disabled-background;
+
+		.counterToggle__icon {
+			color: $disabled-icon;
+		}
+	}
+}
+
+@mixin setCounterToggleAdditions($ripple: null, $icon: null) {
+	@if $ripple == null {
+		.ripple-container {
+			display: none;
+		}
+	}
+
+	.ripple-container {
+		background-color: red !important;
+		//background-color: $ripple !important;
+	}
+
+	.counterToggle {
+		&__icon {
+			color: $icon;
+		}
+	}
+}
+
+.counterToggle {
+	@each $color-name, $color-map in $counter-toggle-colors {
+		&.-color-#{$color-name} {
+			@include setCounterToggleColor(map-get($color-map, 'default', 'color'));
+			@include setCounterToggleBackground(
+				map-get($color-map, 'default', 'background'),
+				map-get($color-map, 'default', 'background-hovered'),
+				map-get($color-map, 'default', 'background-focused')
+			);
+			@include setCounterToggleDisabled(
+				map-get($color-map, 'default', 'disabled', 'color'),
+				map-get($color-map, 'default', 'disabled', 'icon'),
+				map-get($color-map, 'default', 'disabled', 'background')
+			);
+			@include setCounterToggleAdditions(
+				map-get($color-map, 'default', 'ripple'),
+				map-get($color-map, 'default', 'icon')
+			);
+		}
+	}
+
+	align-items: center;
+	flex-direction: row;
+	background-color: transparent;
+	border: 0;
+	border-radius: 4px;
+	cursor: pointer;
+	display: inline-flex;
```

Comment:
Previously counter toggle had a flex display type. Is there a reason for a change?